app/views/layouts/default/application.html.haml

```diff
@@ -24,6 +24,7 @@
 - content_for :head do
   = stylesheet_link_tag 'application'
   = csrf_meta_tags
+  %meta{name:'google-site-verification', content: 'IJ5HOXpZrISM6la-YO_iX0rBUC5YFftpexygcKLsNs4'}
```

Comment:
should this value be coming from config?

Reply:
Not IMO because it will never change, its a one time thing to verify site ownership on Google